docs/api/structures/hid-device.md

```diff
@@ -6,3 +6,11 @@
 * `productId` Integer - The USB product ID.
 * `serialNumber` string (optional) - The USB device serial number.
 * `guid` string (optional) - Unique identifier for the HID interface.  A device may have multiple HID interfaces.
+* `collections` Object[] - an array of report formats. See [Documentation](https://developer.mozilla.org/en-US/docs/Web/API/HIDDevice/collections) for more.
```

Comment:
Since this feature isn't a web standard and is only implemented in Chrome, is there a chance that the spec changes? I see on https://wicg.github.io/webhid/ the following paragraph:

> This specification was published by the [Web Platform Incubator Community Group](https://www.w3.org/groups/cg/wicg). It is not a W3C Standard nor is it on the W3C Standards Track. Please note that under the [W3C Community Contributor License Agreement (CLA)](https://www.w3.org/community/about/agreements/cla/) there is a limited opt-out and other conditions apply. Learn more about [W3C Community and Business Groups](https://www.w3.org/community/).


Would it be beneficial to us to mark it as experimental?

Reply:
@erickzhao that text applies to all WebHID, not `collections`